cmd/preview/list.go

```diff
@@ -73,15 +88,66 @@ func executeListPreviews(ctx context.Context, opts ListFlags) error {
 		return fmt.Errorf("failed to get preview environments: %s", err)
 	}
 	w := tabwriter.NewWriter(os.Stdout, 1, 1, 2, ' ', 0)
-	fmt.Fprintf(w, "Name\tScope\tSleeping\tLabels\n")
-	for _, preview := range previewList {
-		previewLabels := "-"
-		if len(preview.PreviewLabels) > 0 {
-			previewLabels = strings.Join(preview.PreviewLabels, ", ")
+	switch opts.output {
+	case "json":
+		var previewSlice []PreviewJson
```

Comment:
It would be nice to keep it [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) and extract this duplicated code to a func

Reply:
Resolved. I used a little different approach